src/server/removals_reader.rs

```diff
@@ -49,10 +48,7 @@ impl RemovedComponentIds<'_, '_> {
         replication_rules: &ReplicationRules,
     ) -> impl Iterator<Item = (Entity, &[ReplicationId])> {
         self.component_buffer
-            .extend(self.entity_buffer.drain().map(|(_, mut components)| {
-                components.clear();
-                components
-            }));
+            .extend(self.entity_buffer.drain().map(|(_, components)| components));
```

Comment:
I removed `.clear()` in order to be consistent with `client_info` because I have to reset it before the usage.
Since `info` and `entities` are two fields of the same struct, I decided to keep the same behavior.

Reply:
Oh, it's inside `removals_reader`, I missed it. I will revert it.

But keep in `clients_info` since it's necessary.